pkg/util/fedinformer/keys/keys.go

```diff
@@ -111,6 +111,19 @@ func ClusterWideKeyFunc(obj interface{}) (ClusterWideKey, error) {
 	return key, nil
 }
 
+// ClusterWideKeyFuncWithoutGVK generates a ClusterWideKey that without gvk for object.
+func ClusterWideKeyFuncWithoutGVK(obj interface{}) (ClusterWideKey, error) {
+	key := ClusterWideKey{}
+
+	metaInfo, err := meta.Accessor(obj)
+	if err != nil { // should not happen
+		return key, fmt.Errorf("object has no meta: %v", err)
+	}
+	key.Namespace = metaInfo.GetNamespace()
+	key.Name = metaInfo.GetName()
+	return key, nil
+}
```

Comment:
This function does not return a standard `ClusterWideKey`, actually a NamespacedKey.

So, shall we introduce a `NamespacedKey` in addition to `FederatedKey` and `ClusterWideKey`? Like:
```golang
type NamespacedKey struct {
	Namespace string
	Name      string
}
```